src/wdio/services/TerraCommands/match-screenshot-and-test-standards.js

```diff
@@ -0,0 +1,43 @@
+import accessibilityMethods from './accessiblity';
+import visualRegressionMethods from './visual-regression';
+
+const determineOptions = (...args) => {
+  const param1 = args.length ? args[0] : undefined;
+  const param2 = args.length > 1 ? args[1] : undefined;
```

Comment:
Can we add documentation on what we expect the parameters to be before the function declaration?